### PR TITLE
Sys Info Functions

### LINKED ISSUE #143: Document pg_typeof() function
The related PR is https://github.com/singularity-data/risingwave/pull/3494

### PR DESCRIPTION
**Pull request by bernscode**
## Info
- **Description**: 
Support for pg_typeof() function.

- **Related code PR**: 
https://github.com/singularity-data/risingwave/pull/3494

- **Notes**: 
https://github.com/singularity-data/risingwave-docs/issues/143

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked related doc issue to this PR in **Development**.
  - [ ] I have accquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview and the updated parts look all good. <details><summary>How?</summary>image</details>
  - [ ] (For version-specific PR) I have confirmed that this PR is for a released version. If the software version is not released, put it on hold.

## Published doc pages
  [ After merging this PR, edit the description to include the links to the updated doc pages here. For example, https://www.risingwave.dev/docs/latest/intro/. ]

### CODE REVIEW

**Review by hengm3467 — APPROVED** [review]
LGTM

File: sidebars.js [hunk 1]
```diff
@@ -129,6 +129,11 @@ const sidebars = {
             id: 'sql/functions-operators/sql-function-conditional',
             label: 'Conditional',
           },
+          {
+            type: 'doc',
+            id: 'sql/functions-operators/sql-function-sys-info',
+            label: 'System Information',
```
Comment (hengm3467) [inline sidebars.js]: @bernscode System Information - > System information (with "i" in lower case).

File: docs/sql/functions-operators/sql-function-sys-info.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+id: sql-function-sys-info
+slug: /sql-function-sys-info
+title: System information functions
+---
+
+|Function|Description|Example|
+|---|---|---|
+| pg_typeof() → regtype |Returns the standard name of the data type of the provided value. <br /> More specifically, it returns the OID of the data type of the provided value. It returns a regtype, which is an OID alias type. Therefore it’s the same as an OID for comparison purposes but displays as a type name.|`select pg_typeof(round(null));` → `double precision`|
```
Comment (CharlieSYH) [inline docs/sql/functions-operators/sql-function-sys-info.md]: - We don't use complete statements in the examples for functions and operators. Therefore: `pg_typeof(round(null))` → `double precision`
- Could have more examples.

**Comment by CharlieSYH** [comment]
Now that the standard names for data types are finalized, we can update and backfill the Data types page https://www.risingwave.dev/docs/latest/sql-data-types/. @bernscode 

             DataType::Boolean => f.write_str("boolean"),
             DataType::Int16 => f.write_str("smallint"),
             DataType::Int32 => f.write_str("integer"),
             DataType::Int64 => f.write_str("bigint"),
             DataType::Float32 => f.write_str("real"),
             DataType::Float64 => f.write_str("double precision"),
             DataType::Decimal => f.write_str("numeric"),
             DataType::Date => f.write_str("date"),
             DataType::Varchar => f.write_str("character varying"),
             DataType::Time => f.write_str("time without time zone"),
             DataType::Timestamp => f.write_str("timestamp without time zone"),
             DataType::Timestampz => f.write_str("timestamp with time zone"),
             DataType::Interval => f.write_str("interval"),
             DataType::Struct { .. } => f.write_str("record"),
             DataType::List { datatype } => write!(f, "{}[]", datatype),

**Review by CharlieSYH — CHANGES_REQUESTED** [review]

**Comment by CharlieSYH** [comment]
Please set the Milestone for your other pull requests. I've selected the correct version for this PR:
image
You can get the version in the corresponding doc issue.

File: sidebars.js (hunk 1 above)
Comment (bernscode) [reply sidebars.js]: Updated and committed change. Thanks!

File: docs/sql/functions-operators/sql-function-sys-info.md (hunk 2 above)
Comment (bernscode) [reply docs/sql/functions-operators/sql-function-sys-info.md]: Nice catch! I noticed this while working through other updates. Updated and committed change. Thanks!

**Review by CharlieSYH — APPROVED** [review]

**Pull request merged**